### PR TITLE
Sort HTTP methods to remove randomness from keys

### PR DESCRIPTION
**Pull request by Chewie69006**
Hello !

As `data.keys` returns in a random order, comparing 2 arrays containing same objects in different order make it return false. This way, it will return true as they will be sorted.

### CODE REVIEW

**Review by AvdLee — CHANGES_REQUESTED** [review]
Great catch! Would it be possible to write a unit test for this as well? That way, we prevent it from happening again in the future.

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Consider to place some `MARK:` lines for Sources/Mocker/Mock.swift, which is over 300 lines big.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/7a22268e-321f-4a7c-84aa-859ba4cf6d7e" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>

### SwiftLint found issues

| Severity | File | Reason |
| -------- | ---- | ------ |
Warning | MockTests.swift:10 | Imports should be sorted (`sorted_imports`) |
Warning | MockTests.swift:35 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | MockTests.swift:38 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:9 | The disabled 'force_unwrapping' rule should be re-enabled before the end of the file (`blanket_disable_command`) |
Warning | Mock.swift:93 | Computed properties should first declare the getter and then the setter (`computed_accessors_order`) |
Error | Mock.swift:111 | Prefer checking `isEmpty` over comparing `count` to zero (`empty_count`) |
Error | Mock.swift:8 | Line should be 160 characters or less; currently it has 177 characters (`line_length`) |
Error | Mock.swift:38 | Line should be 160 characters or less; currently it has 195 characters (`line_length`) |
Warning | Mock.swift:55 | Line should be 140 characters or less; currently it has 155 characters (`line_length`) |
Error | Mock.swift:87 | Line should be 160 characters or less; currently it has 211 characters (`line_length`) |
Error | Mock.swift:90 | Line should be 160 characters or less; currently it has 208 characters (`line_length`) |
Error | Mock.swift:101 | Line should be 160 characters or less; currently it has 217 characters (`line_length`) |
Error | Mock.swift:110 | Line should be 160 characters or less; currently it has 297 characters (`line_length`) |
Warning | Mock.swift:116 | Line should be 140 characters or less; currently it has 152 characters (`line_length`) |
Warning | Mock.swift:156 | Line should be 140 characters or less; currently it has 141 characters (`line_length`) |
Error | Mock.swift:159 | Line should be 160 characters or less; currently it has 171 characters (`line_length`) |
Warning | Mock.swift:178 | Line should be 140 characters or less; currently it has 141 characters (`line_length`) |
Warning | Mock.swift:185 | Line should be 140 characters or less; currently it has 142 characters (`line_length`) |
Error | Mock.swift:186 | Line should be 160 characters or less; currently it has 246 characters (`line_length`) |
Warning | Mock.swift:204 | Line should be 140 characters or less; currently it has 141 characters (`line_length`) |
Error | Mock.swift:206 | Line should be 160 characters or less; currently it has 171 characters (`line_length`) |
Error | Mock.swift:211 | Line should be 160 characters or less; currently it has 256 characters (`line_length`) |
Warning | Mock.swift:234 | Line should be 140 characters or less; currently it has 150 characters (`line_length`) |
Error | Mock.swift:249 | Line should be 160 characters or less; currently it has 171 characters (`line_length`) |
Warning | Mock.swift:253 | Line should be 140 characters or less; currently it has 160 characters (`line_length`) |
Warning | Mock.swift:268 | Line should be 140 characters or less; currently it has 141 characters (`line_length`) |
Error | Mock.swift:270 | Line should be 160 characters or less; currently it has 171 characters (`line_length`) |
Error | Mock.swift:275 | Line should be 160 characters or less; currently it has 262 characters (`line_length`) |
Warning | Mock.swift:326 | Line should be 140 characters or less; currently it has 159 characters (`line_length`) |
Error | Mock.swift:336 | Line should be 160 characters or less; currently it has 168 characters (`line_length`) |
Warning | Mock.swift:129 | Use shorthand syntax for optional binding (`shorthand_optional_binding`) |
Warning | Mock.swift:338 | Use shorthand syntax for optional binding (`shorthand_optional_binding`) |
Warning | Mock.swift:338 | Use shorthand syntax for optional binding (`shorthand_optional_binding`) |
Warning | Mock.swift:42 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:114 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:155 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:199 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:224 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:244 | Lines should not have trailing whitespace (`trailing_whitespace`) |
Warning | Mock.swift:263 | Lines should not have trailing whitespace (`trailing_whitespace`) |
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against e08d07a249f1f582b617c74b0677d92edb8cef44
</p>

**Review by AvdLee — APPROVED** [review]

**Comment by AvdLee** [comment]
@Chewie69006 thanks for writing the test, great job! Let's wait for one more review and we can merge.

**Review by BasThomas — APPROVED** [review]
Great!

**Comment by BasThomas** [comment]
Looks like your commit adding the test isn't signed, which blocks us from merging. Can you take care of that, @Chewie69006? Let us know if we can help.

**Comment by Chewie69006** [comment]
When do you think a new tag will be delivered ?

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 3.0.2](https://github.com/WeTransfer/Mocker/releases/tag/3.0.2) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**
